plugins/linux/ifplugin/data_resync.go

```diff
@@ -238,7 +238,7 @@ func (plugin *LinuxInterfaceConfigurator) isLinuxIfModified(nbIf, linuxIf *inter
 	}
 	// Remove IPv6 link local addresses (default values)
 	for ipIdx, ipAddress := range linuxIf.IpAddresses {
-		if strings.HasPrefix(ipAddress, "fe80") {
+		if strings.HasPrefix(ipAddress, nsplugin.IPv6LinkLocalPrefix) {
```

Comment:
This should rather use [existing API from standard library](https://godoc.org/net#IP.IsLinkLocalUnicast) to correctly check if the IP address is link local unicast.